lib/pages/settings/model/request.dart

```diff
@@ -1,9 +1,18 @@
 class Request {
-  Request(this.userId, this.requestBody);
+  Request(this.userId, this.requestBody,
```

Comment:
Why not make all parameters named? It's generally the standard way to do it in Dart. You can annotate them as @required if you always want them specified. I'd also make the type required and not give it a default value.